server/middleware/authorization.js

```diff
@@ -0,0 +1,26 @@
+const jwt = require('jsonwebtoken');
+require('dotenv').config({path: "./vars/.env"});
+
+const authorization = (req,res,next)=>{
+
+    const jwtToken = req.header("token");
+
+    if(!jwtToken){
+        return res.status(403).json('Not Authorize');
```

Comment:
Status code `401` should be returned.

Reply:
The client does not have access rights to the content; that is, it is unauthorized, so the server is refusing to give the requested resource. Unlike 401 Unauthorized, the client's identity is known to the server.

Reply:
As things stand I see that jwtToken can be null from line 6. That means that we don't have the token, therefore user is unknown?

Reply:
Yeah, you're right. Client's identity is known but in case token is null he just don't have right's to access resourses.

Reply:
https://www.freecodecamp.org/news/http-401-error-vs-http-403-error-status-code-responses-explained/
Read it very carefully, YOU CANNOT HAVE 403 on auth endpoints, on those endpoints. Just no logic in that. Because token is the single source of truth in case of auth. And when we don't have token we don't know shit.
Here we don't have token, that means we don't know the user. You cannot forbid someone to enter your birthday party before knowing who they are :)